doc/news/_preparation_next_release.md

```diff
@@ -212,7 +212,7 @@ _(Michael Tucek)_
 
 ## Packaging
 
-- <<TODO>>
+- add flatpak package _(Ivaylo Ivanov)_
```

Comment:
Please add link to scripts/flatpak and ideally the command to directly pull your package from flathub.

Reply:
I mentioned where to find more info about building the package. 
About Flathub: we need to go through an [approval process](https://github.com/flathub/flathub/wiki/App-Submission
) first or we can host our own flatpak repo. I think that both of these are out of scope for this issue and should be tackled when the package is fully functional.